perun-base/src/main/java/cz/metacentrum/perun/core/api/Role.java

```diff
@@ -25,10 +29,10 @@
 	public static final String SECURITYADMIN = "SECURITYADMIN";
 	public static final String CABINETADMIN = "CABINETADMIN";
 	public static final String UNKNOWNROLENAME = "UNKNOWN";
-	
+
 	public static List<String> rolesAsList() {
-		return Arrays.asList(CABINETADMIN, ENGINE, FACILITYADMIN, GROUPADMIN,
-			 NOTIFICATIONS, PERUNADMIN, PERUNOBSERVER, REGISTRAR, RESOURCEADMIN, RESOURCESELFSERVICE,
+		return Arrays.asList(CABINETADMIN, ENGINE, FACILITYADMIN, FACILITYOBSERVER, GROUPADMIN, GROUPOBSERVER,
+			 NOTIFICATIONS, PERUNADMIN, PERUNOBSERVER, REGISTRAR, RESOURCEADMIN, RESOURCEOBSERVER, RESOURCESELFSERVICE,
```

Comment:
Please add the TUSTEDFACILITYADMIN also to this method.

Reply:
Oh sorry, I forgot about it. Fixed!